sdks/java/io/thrift/src/test/java/org/apache/beam/sdk/io/thrift/TestThriftStruct.java

```diff
@@ -0,0 +1,1232 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
It can be generated from the .thrift file that is included. I think we would need to add a thrift compiler to the build.gradle to compile it for testing, thoughts?

Reply:
Hm in that case, I think it's fine to commit the generated file.

Reply:
I don't think that thrift guarantees that generated code is forwards or backwards compatible, meaning that if we upgrade libthrift here, the code might no longer compile and need to be regenerated.  I'm +1 for having this being generated at compile time.

Reply:
Sounds good, I'll work on this.